README.md

```diff
@@ -50,6 +54,96 @@ graph TD
+    E[Flags joined to<br>default.vw_pin_sale]
+
+    A -- Some sales are not flagged --> B
+    B --> C
+    C --> D
+    D -- Results saved to S3<br>with unique run ID --> E
+
+
+```
+
+# Flags at a Glance
+
+Sales from 2014 - present have been processed using our sales validation program. We assign flags to the following classes of properties:
+
+- **Residential** - 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212, 218, 219, 234, 278, 295
+- **Condos** - 297, 299, 399
+
+## Outlier Types
+
+In order to be flagged as on outlier type, the property needs to be a statistical price outlier. A statistical price outlier is a sale price that is some number of standard deviations outside a grouping of similar properties (eg. township, class, time frame).  However, there are also special flags that combine with the statistical outlier type, these sales *should* be more likely to be non-arms length sales than the regular price outlier sales. Examples of these special flags are:
+- **PTAX flag**: The PTAX-203 form is required to be filled out for an Illinois Real Estate Transfer Declaration for non-residential property over $1 million and/or as required by Illinois Department of Revenue. If there are certain fields filled out on this form, we mark the sale with a ptax flag. 
+- **Non-person sale**: We flag a keyword that suggests the sale involves a legal entity (industrial buyer, bank, real estate firm, construction, etc)
+- **Flip Sale**: The owner of the home owned the property for less than 1 year
+- **Anomaly**: Our isolation forest statistical outlier algorithm flagged the sale
+
+The following is a list of all flag types:
+
+### High Price
+
+- **PTAX outlier (high)**: PTAX flag & [1 high statistical outlier type]
+- **Home flip sale (high)**: Short-term owner < 1 year & [1 high statistical outlier type]
+- **Family sale (high)**: Last name match & [1 high statistical outlier type]
+- **Non-person sale (high)**: Legal / corporate entity + [1 high statistical outlier type]
+- **Anomaly (High)**: Anomaly algorithm (high) & [1 high statistical outlier type]
+- **High price (raw & sqft )**: High price & high price per sq. ft. 
+- **High price swing**: Large swing away from mean  & high price outlier 
+- **High price (raw)**: High price 
+- **High price (per sqft)**: High price per sq. ft. 
+
+### Low Price
+
+- **PTAX outlier (high)**: PTAX flag & [1 low statistical outlier type]
+- **Home flip sale (low)**: Short-term owner < 1 year & [1 low statistical outlier type]
+- **Family sale (low)**: Last name match & [1 low statistical outlier type]
+- **Non-person sale (low)**: Legal / corporate entity + [1 low statistical outlier type]
+- **Anomaly**: - Anomaly algorithm (low) & [1 low statistical outlier type]
+- **Low price (raw & sqft )**: Large swing away from mean & low price outlier 
+- **Low price swing**: Low price & low price per sq. ft. 
+- **Low price (raw)**: Low price (or under $10k)
+- **Low price (per sqft)**: Low price per sq. ft.
```

Comment:
**suggestion:** I recommend you convert all of this into a markdown table. Right now it's difficult to read.